pkg/security/security.go

```diff
@@ -131,11 +132,28 @@ type Options struct {
 	// Location of JWTPath to connect to CA.
 	JWTPath string
 
-	// OutputKeyCertToDir is the directory for output the key and certificate
+	// OutputKeyCertToDir (env: OUTPUT_CERTS) is the directory for output the key and certificate.
+	// If not set, the private key and cert will be in memory and made available to Envoy using SDS.
+	//
+	// For VMs must be set to the same value with PROV_CERT to enable auto-refresh of the certificate.
```

Comment:
Can we merge the two environment variables into one? They should always be set to the same values, correct?

Reply:
No, there is a use case where PROV_CERTS is set to a long lived cert (provisioned externally), that is not rotated by Istiod.

I just added comments in this PR - we can discuss making changes in separate PR but not related to STS ( comments are probably not very related either, but I spent some time looking at this file to understand and it seemed undercommented)